Sources/Rendering/Misc/SynchronizableRenderWindow/vtkObjectManager.js

```diff
@@ -116,9 +116,9 @@ function update(type, instance, props, context) {
   }
   const handler = TYPE_HANDLERS[type];
   if (handler && handler.update) {
-    handler.update(instance, props, context);
+    return handler.update(instance, props, context);
   } else {
-    console.log('no updater for', type);
+    return Promise.reject(new Error('No updater for ${type}'));
```

Comment:
To have `${type}` properly templated you need to use the back tick rather than the standard '.